olive/passes/onnx/transformer_optimization.py

```diff
@@ -97,6 +97,23 @@ def validate_search_point(
         if search_point.get("use_gpu") and accelerator_spec.execution_provider == "CPUExecutionProvider":
             logger.info("CPUExecutionProvider does not support GPU inference, please avoid to use use_gpu.")
             return False
+        if search_point.get("only_onnxruntime") and search_point.get("opt_level") <= 0:
+            logger.info("Please specify a positive value for opt_level when only_onnxruntime is True")
+            return False
+        if (
+            search_point.get("opt_level") == 0
+            and search_point.get("only_onnxruntime")
+            and search_point.get("num_heads") == 0
+            and search_point.get("hidden_size") == 0
```

Comment:
seems the ort optimize_by_fused allow 0 for bert model?

Reply:
Yes, opt_level can be 0, but at the same time the num_heads and hidden_size cannot be 0 as this issue:
https://github.com/microsoft/onnxruntime/issues/17254